src/schema/objects/enums.yaml

```diff
@@ -1087,3 +1103,305 @@ NIRSCWMUA:
+control:
+  value: control
+  display_name: control
+  description: |
+    An ASL volume type: control
+label:
+  value: label
+  display_name: label
+  description: |
+    An ASL volume type: label
+m0scan:
+  value: m0scan
+  display_name: m0scan
+  description: |
+    An ASL volume type: m0scan
+deltam:
+  value: deltam
+  display_name: deltam
+  description: |
+    An ASL volume type: deltam
+cbf:
+  value: cbf
+  display_name: cbf
+  description: |
+    An ASL volume type: cbf
+single_coil:
+  value: single-coil
+  display_name: single-coil
+  description: |
+    CASL Type: when a single coil is used for labeling.
+double_coil:
+  value: double-coil
+  display_name: double-coil
+  description: |
+    CASL Type: when a double coil is used for labeling.
+IODINE:
+  value: IODINE
+  display_name: IODINE
+  description: |
+    A Contrast Bolus Ingredient: Iodine
+GADOLINIUM:
+  value: GADOLINIUM
+  display_name: GADOLINIUM
+  description: |
+    A Contrast Bolus Ingredient: Gadolinium
+CARBON_DIOXIDE:
+  value: CARBON DIOXIDE
+  display_name: CARBON DIOXIDE
+  description: |
+    A Contrast Bolus Ingredient: Carbon Dioxide
+BARIUM:
+  value: BARIUM
+  display_name: BARIUM
+  description: |
+    A Contrast Bolus Ingredient: Barium
+XENON:
+  value: XENON
+  display_name: XENON
+  description: |
+    A Contrast Bolus Ingredient: Xenon
+raw:
+  value: raw
+  display_name: raw
+  description: |
+    A raw BIDS dataset.
+derivative:
+  value: derivative
+  display_name: derivative
+  description: |
+    A derived BIDS dataset.
+balanced:
+  value: balanced
+  display_name: balanced
+  description: |
+    PCASL Type: when balanced gradient pulses are used.
+unbalanced:
+  value: unbalanced
+  display_name: unbalanced
+  description: |
+    PCASL Type: when unbalanced gradient pulses are used.
+left_hand:
+  value: left-hand
+  display_name: left-hand
+  description: |
+    A rotation rule: rotations are applied clockwise around
+    an axis when seen from the positive direction
+right_hand:
+  value: right-hand
+  display_name: right-hand
+  description: |
+    A rotation rule: rotations are applied counter-clockwise around
+    an axis when seen from the positive direction
+in_vivo:
+  value: in vivo
+  display_name: in vivo
+  description: |
+    The environment of a sample: in vivo
+ex_vivo:
+  value: ex vivo
+  display_name: ex vivo
+  description: |
+    The environment of a sample: ex vivo
+in_vitro:
+  value: in vitro
+  display_name: in vitro
+  description: |
+    The environment of a sample: in vitro
+blood:
+  value: blood
+  display_name: blood
+  description: |
+    The origin of a sample: blood
+saliva:
+  value: saliva
+  display_name: saliva
+  description: |
+    The origin of a sample: saliva
+brain:
+  value: brain
+  display_name: brain
+  description: |
+    The origin of a sample: brain
+csf:
+  value: csf
+  display_name: csf
+  description: |
+    cerebrospinal fluid (e.g., the origin of a sample or tissue)
```

Comment:
```suggestion
  value: csf
  display_name: CSF
  description: |
    The origin of a sample: cerebrospinal fluid
```

Reply:
I left this a bit more ambiguous than the other enum descriptions because `csf` is used in two different places (once for sample, once for tissue)

Reply:
I see. I was conforming this because the Latin phrases CI was complaining.